protocol/superc20/mirror-standard.md

```diff
@@ -0,0 +1,114 @@
+# Mirror Standard Design Doc
+
+## Purpose
+
+_The work present below is part of the [Interoperability project](https://github.com/ethereum-optimism/optimism/issues/10899)._
+
+This document presents the `SuperchainERC20` Mirror Standard, which enables existing tokens deployed through `OptimismMintableERC20` Factory in L2 (also called legacy tokens) to be interoperable without any liquidity migration process.
+
+## Summary
+
+We discussed different approaches to making existing liquidity interoperable in the [Migrated Liquidity Design Doc](https://github.com/ethereum-optimism/design-docs/pull/35). We introduced the Mirror standard and highlighted it as the simplest solution, as it requires no liquidity migration at all.
+
+The Mirror approach deploys a new token contract that follows the `ISuperchainERC20` specifications and will work as a double-entry point for an appointed `OptimismMintableERC20`. We must modify the `L2StandardBridge` to enable the Mirror contract to burn and mint the underlying `OptimismMintableERC20` to implement the Mirror.
+
+## Proposed Design
+
+The proposed design includes a specification on implementing a double-entry point to mirror an existing `OptimismMintableERC20`, as well as approaches to the Factory and `L2StandardBridge`.
+
+### OptimismMintableERC20 proxiable functions
+
+The following functions will be implemented as simple proxies, that is, by having a mirror call to the appointed `OptimismMintableERC20` token:
+
+- `totalSupply()`
+- `balanceOf(account)`
+- `transfer(recipient, amount)`
+- `transferFrom(sender, recipient, amount)`
+
+### Local functions
+
+Approvals functions will be implemented in a conventional way, specific to the contract, and not as mirror calls:
+
+- `approve(spender, amount)`
+- `allowance(owner, spender)`
+
+### Code sample
+
+Below, we show what the implementation of the mirror contract would look like.
+
+```solidity
+contract Mirror is Superc20 {
```

Comment:
Since this references the design in https://github.com/ethereum-optimism/specs/pull/71, can we assume general consensus on the design in that PR?

Reply:
> an we assume general consensus on the design in that PR?

I was going to ask a similar question, and I think the answer is no we cannot assume general consensus, since that PR is a bit stale with a lot of open comments. It feels like that PR is a prerequisite for this one + interoperable ether (https://github.com/ethereum-optimism/design-docs/pull/25), so I would suggest reprioritizing the SuperchainERC20 standard so we can finalize that first

Reply:
Yes. The mirror design will require a particular implementation for the cross-chain actions, but we can assume here no compatibility issues.